includes/admin/payments/orders.php

```diff
@@ -742,33 +794,60 @@ function edd_order_details_extras( $order = false ) {
 						<span class="label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></span>
 						<span class="value"><?php echo edd_get_gateway_admin_label( $order->gateway ); ?></span>
 					</div>
-					<?php endif; ?>
 				<?php else : ?>
 					<div class="edd-order-gateway edd-admin-box-inside">
-						<span class="label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></span>
-						<?php echo $gateway_select; ?>
+						<div class="edd-form-group">
+							<label for="edd_gateway_select" class="edd-form-group__label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></label>
+							<div class="edd-form-group__control">
+								<?php
+								echo EDD()->html->select(
+									array(
+										'name'             => 'gateway',
+										'class'            => 'edd-form-group__input',
+										'id'               => 'edd_gateway_select',
+										'options'          => $gateways,
+										'selected'         => edd_get_default_gateway(),
+										'chosen'           => true,
+										'show_option_none' => false,
+										'show_option_all'  => false,
+									)
+								); // WPCS: XSS ok.
+								?>
+							</div>
+						</div>
 					</div>
 				<?php endif; ?>
 
-				<div class="edd-order-payment-key edd-admin-box-inside">
-					<label for="edd_payment_key" class="label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
-					<input type="text" name="payment_key" id="edd_payment_key" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+				<div class="edd-admin-box-inside">
+					<div class="edd-form-group">
+						<label for="edd_payment_key" class="edd-form-group__label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
+						<div class="edd-form-group__control">
+							<input type="text" name="payment_key" id="edd_payment_key" class="edd-form-group__input" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+						</div>
+					</div>
 				</div>
 
-				<div class="edd-order-ip edd-admin-box-inside">
-					<label for="edd_ip" class="label"><?php esc_html_e( 'IP', 'easy-digital-downloads' ); ?></label>
-					<?php if ( edd_is_add_order_page() ) : ?>
-						<input type="text" name="ip" id="edd_ip" value="<?php echo esc_attr( edd_get_ip() ); ?>" />
-					<?php else : ?>
-						<span><?php echo edd_payment_get_ip_address_url( $order->id ); // WPCS: XSS ok. ?></span>
-					<?php endif; ?>
-				</div>
+				<?php if ( edd_is_add_order_page() ) : ?>
+					<div class="edd-order-ip edd-admin-box-inside">
+						<div class="edd-form-group">
+							<label for="edd_ip" class="edd-form-group__label"><?php esc_html_e( 'IP', 'easy-digital-downloads' ); ?></label>
+							<div class="edd-form-group__control">
+								<input type="text" name="ip" id="edd_ip" class="edd-form-group__input" readonly value="<?php echo esc_attr( edd_get_ip() ); ?>" />
```

Comment:
This shouldn't be `readonly`. It should be prefilled, but with the ability to edit it.